lab_7_llm/main.py

```diff
@@ -6,9 +6,13 @@
 from pathlib import Path
```

Comment:
Checkout the files from 7 lab to the master

Reply:
done